service/tasks/driver.py

```diff
@@ -32,7 +32,7 @@
 from core.models.identity import Identity
 from core.models.profile import UserProfile
 
-from service.deploy import check_process, wrap_script, echo_test_script,\
+from service.deploy import _inject_env_script, check_process, wrap_script, echo_test_script,\
```

Comment:
If we _intend_ for `_inject_env_script` to be used outside of the defining file, I would change this to `inject_env_script`.